lib/src/signed_video_h26x_sign.c

```diff
@@ -296,6 +296,7 @@ generate_sei_nalu(signed_video_t *self, uint8_t **payload, uint8_t **payload_sig
     // reserved_byte = |epb|golden sei|0|0|0|0|0|0|
     uint8_t reserved_byte = self->sei_epb << 7;
     reserved_byte |= self->is_golden_sei << 6;
+    reserved_byte |= !self->gop_hash_off << 4;
```

Comment:
`gop_hash_off` is only used here, hence there is no need to have a variable for it.
```
reserved_byte |= 1 << 4;  // Flag to validation side that this stream computes the GOP hash differently
```

Reply:
I was thinking it can also help to have it while validating.

Reply:
OK. Fair enough.